BUILDING.md

```diff
@@ -85,6 +85,17 @@ or setting the value upon execution of the `hab` binary:
 env HAB_SUP_BINARY=/path/to/habitat/target/debug/hab-sup hab sup status
 ```
 
+## HAB_STUDIO_BINARY
+
+This one is a bit special. Technically [hab-studio.sh](https://github.com/habitat-sh/habitat/blob/studio-testing-comment/components/studio/bin/hab-studio.sh) is a shell script file and not binary. This also means that there is no need to build anything; set `HAB_STUDIO_BINARY` to the path to a version of `hab-studio.sh` within a `habitat` checkout and it will be used. This override will also affect which versions of the files in [studio/libexec](https://github.com/habitat-sh/habitat/tree/studio-testing-comment/components/studio/libexec) are used. So if you want to test out changes to [hab-studio-profile.sh](https://github.com/habitat-sh/habitat/blob/studio-testing-comment/components/studio/libexec/hab-studio-profile.sh) or [hab-studio-type-default.sh](https://github.com/habitat-sh/habitat/blob/studio-testing-comment/components/studio/libexec/hab-studio-type-default.sh), make those changes in a checkout of the `habitat` repo located at `/path/to/habitat/repo` and set `HAB_STUDIO_BINARY` to `/path/to/habitat/repo/components/studio/bin/hab-studio.sh`. For example:
+```bash
```

Comment:
You probably want to change the links in this block of text to be something that's not `https://github.com/habitat-sh/habitat/blob/studio-testing-comment/components/*`, since these links will all 404 after https://github.com/habitat-sh/habitat/pull/4825 merges and the `studio-testing-comment` branch is deleted.

Reply:
Good catch! Thank you.